scene/gui/scroll_container.cpp

```diff
@@ -316,6 +316,8 @@ void ScrollContainer::_notification(int p_what) {
 			}
 			r.position += ofs;
 			fit_child_in_rect(c, r);
+			// move the child to the top of the list, so the scrollbars are displayed on top
+			move_child(c, 0);
```

Comment:
still kind of hack but it would be better `move_child(c, get_child_count() - 3);`

Reply:
or `move_child(c, get_child_count() - get_child_count());` to be future proof (in case someone adds a new "inner" child to scrollcontainer...) ?

Reply:
`get_child_count() - get_child_count()` that will always be 0 :)

Reply:
[DontHitMeItsJustHumour]
@akien-mga you're so smart 😆 
[/DontHitMeItsJustHumour]
yep that's why I put directly 0 in the code
but sometimes people prefer to read a variable name instead of a simple number

Reply:
But that doesn't make it "future proof" (unless that was also part of the joke, in which case it was lost on me :)).

In any case it's a hack, but if the intent is to have this as the first child, then 0 is indeed the right way to do it.

Reply:
I think it would be better to move the scrollbar children at the bottom instead. So that if you add several children they would in the same order, even if not added at the end.